README.md

```diff
@@ -17,8 +17,32 @@ Debain, Ubuntu, and derivatives are most supported for builds. To build, you nee
 apt install curl build-essential git autoconf automake libtool bison flex gcc ruby rake bundler git mingw-w64
 ```
 
-The Dockerfile under docker/Dockerfile contains a pre-configured build
-environment as well.
+Building on Docker
+------
+
+You can download a pre-built Docker image from [Rapid7's Docker Hub account](https://hub.docker.com/u/rapid7):
+```
+docker pull rapid7/msf-ubuntu-x64-meterpreter:latest
```

Comment:
Looks like we're not using this, and we're relying on `rapid7/build:mettle`

That aside, the current Dockerfile isn't quite enough to run standalone without mounted volumes. Will close for now.